### PR TITLE
Multi-arch support

### PR DESCRIPTION
**Pull request by jornane**
There is a Windows version for ARM and ARM64. Microsoft has made it possible to run 32-bit Intel binaries on ARM platforms, but not 64-bit Intel binaries. This is a problem because 32-bit binaries for geteduroam do not work fa826f3, so we need to build different versions of the application.

### CODE REVIEW

**Comment by pbsds** [comment]
Looking good!
This PR is missing WpfApp, which is the new GUI currently being developed.
We could also remove x86 entirely while at it.
Would be nice with some ARM hardware to test with, perhaps a RPi would suffice.

I have a slight fear this might not work since we [link with win32 DLLs](https://github.com/geteduroam/eduroam-supplicant-win/blob/main/ManagedNativeWifi/Win32/NativeMethod.cs), and i fear this might have moved on ARM.

File: EduroamApp.sln [hunk 1]
```diff
@@ -14,57 +14,93 @@ EndProject
...
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM = Debug|ARM
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|ARM = Release|ARM
+		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
```
Comment (pbsds) [inline EduroamApp.sln:25]: Be gone x86!

**Comment by pbsds** [comment]
Documenting what you need to install in visual studio would also be nice

**Comment by jornane** [comment]
The app works in emulation mode, but this PR doesn't really seem to work for compiling to ARM.

**Pull request closed without merging**